src/porepy/numerics/fv/tpfa.py

```diff
@@ -69,26 +69,19 @@ def discretize(self, g, data, vector_source=False, faces=None):
         ----------
         g (pp.Grid): grid, or a subclass, with geometry fields computed.
         data (dict): For entries, see above.
-        vector_source (Boolean): optional. Defines flux contribution from vector source
-        faces (np.ndarray): optional. Defines active faces.
```

Comment:
Do we want to provide a summary of the keywords of data (or rather of data["parameters"]) which are accessed by the discretize method? I.e. "second_order_tensor" etc

Reply:
It is already there, in the discretize() method.